.github/workflows/release_build.yaml

```diff
@@ -191,7 +193,9 @@ jobs:
           NUM_RUNNERS: ${{ matrix.num_runners }}
           THIS_RUNNER: ${{ matrix.runner_id }}
           TERM: dumb
-          CICD_TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
+          # We don't need to specify CICD_TARGET_BRANCH since the upgrade
+          # integration test will detect the annotated tag for version checking.
+          # CICD_TARGET_BRANCH:
```

Comment:
Just to be clear, we don't need this here because this is not a PR build? `02-verify-if-codebase-is-updated` has the following to say:
```sh
    # Determine which branch to detect the "latest" version from:
    # - for PRs, this will be the branch the PR targets (as supplied via
    #   CICD_TARGET_BRANCH by the CI/CD pipeline).
    # - for non-PRs from a local branch with a tracking branch, we'll use
    #   the tracking branch (e.g. local development branch tracking master)
    # - for non-PRs from a local branch without a tracking branch, we'll fail
    #   the test, since it isn't clear which version we should be tracking.
```

Reply:
If you look farther down that script you'll see where we handle the case when we're testing against an annotated tag:

https://github.com/spiffe/spire/blob/8b52c49bb76ee96360246e19a9c04b2b28a30305/test/integration/suites/upgrade/02-verify-codebase-version-is-updated#L59